src/olympia/activity/admin.py

```diff
@@ -38,9 +46,33 @@ def has_add_permission(self, request):
     def has_delete_permission(self, request, obj=None):
         return False
 
+    @admin.display(description='Arguments')
+    def pretty_arguments(self, obj):
+        return obj.to_string(type_='admin')
```

Comment:
TODO: Need to add a test proving that this works and that it doesn't escapes the HTML content

Reply:
https://github.com/mozilla/addons-server/pull/20246/commits/314ba482fcef817940fd285a15116ee40fcc9361